load_tests/load_test.py

```diff
@@ -16,7 +16,7 @@
 PREFIX = os.environ['PREFIX']
 EKS_CLUSTER_NAME = os.environ['EKS_CLUSTER_NAME']
 LOGGER_RUN_TIME_IN_SECOND = 600
-BUFFER_TIME_IN_SECOND = 300
+BUFFER_TIME_IN_SECOND = 1000
```

Comment:
Why we increase the buffer time ?

Reply:
See the comment on the other file. This comes from recent testing, if you call GetLogEvents to quickly after a put, you get inconsistent results. The longer you wait, the more likely eventual consistency will not hurt you. 

If you call at too high of a TPS, you also can get inconsistent results.